pkgs/applications/qubes/qubes-core-vchan-xen/default.nix

```diff
@@ -24,13 +24,17 @@ stdenv.mkDerivation rec {
     make install DESTDIR=$out PREFIX=/
   '';
 
+  # FIXME: Should only be enabled on xen 4.18+. It is also possible to make this detection in makefile,
```

Comment:
I'll hardcode some of similar version check results, I don't think it is worth to support those.

Still it would be better if upstream performed those checks using pkg-config, and not using `ls /usr/includes/xen/xs.h` etc